benchmarks/ner_embeddings/scripts/custom_components.py

```diff
@@ -0,0 +1,2 @@
+from scripts.multiembed import *
```

Comment:
So the way I understand it is that this file will be the one we pass through the `--code` parameter when training. Is it possible if we just import the relevant classes instead of `import *`?